packages/frontapp/src/app/components/NotFoundWindow/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC } from 'react';
+import styled from '@emotion/styled';
+import { colors, dimensions } from '@mimir/ui-kit';
+
+const NotFoundWindowWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: ${colors.bg_secondary};
+  margin: 30px 0px 30px 0px;
+  max-width: 1015px;
+  width: 100%;
+  height: 448px;
+  border-radius: 10px;
+  box-shadow: 0px -4px 64px rgba(24, 39, 75, 0.12);
+
+  @media (max-width: ${dimensions.tablet_width}) {
+    width: 100%;
+    height: 336px;
+    background: #ffffff;
+    box-shadow: 0px -4px 64px rgba(24, 39, 75, 0.12);
+    border-radius: 10px;
+  }
+`;
+
+const WarningMessage = styled.div`
+  width: 508px;
+  height: 76px;
```

Comment:
Any reason to set height? If someone wants to change font-size it will not look good